.github/dependabot.yml

```diff
@@ -0,0 +1,8 @@
+version: 2
+updates:
+  # Maintain dependencies for GitHub Actions
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      # Check for updates to GitHub Actions every week
+      interval: "weekly"
```

Comment:
While this looks fairly straightforward, since I typically attend both the NumPy and SciPy meetings I did want to point out that NumPy has had some pretty bad experiences with dependabot, to the point that they removed it: https://github.com/numpy/numpy/pull/20268

See also:
https://github.com/numpy/numpy/issues/18977

And the same version of this PR in NumPy repo:
https://github.com/numpy/numpy/pull/22514

My experience is that I only bother updating the actions ~once per year, and it rarely makes much difference, so I'm not sure we gain much with automation vs. having new infrastructure just to bump it.

Perhaps if you convince the NumPy team they should re-add it after all their bad experiences we could consider..